src/Domain/Block.cpp

```diff
@@ -131,11 +133,14 @@ void Block<VolumeDim>::pup(PUP::er& p) {
     p | neighbors_;
     p | external_boundaries_;
   }
+  if (version >= 1) {
+    p | name_;
```

Comment:
Maybe I'm just missing it, but is there an `==` that should have this for Block too?

Reply:
ugh thanks for catching this too 🤦‍♂️